### PR TITLE
doc: Link directly to interactive notebooks with less UI clutter.

### PR DESCRIPTION
**Pull request by PapyChacal**

### CODE REVIEW

**Comment by tobiasgrosser** [comment]
There is a typo in the commit message. It's 'directly'.

File: README.md [hunk 1]
```diff
@@ -35,12 +35,10 @@ pip install xdsl
 
 ## Using xDSL
```
Comment (georgebisbas) [inline README.md:37]: We could add some text here...

XDSL...can be useful for A and B....The following tutos....can help to...educate....etc

Comment (tobiasgrosser) [reply README.md:37]: That's a great idea. @PapyChacal, do you want to add such a sentence?

Comment (tobiasgrosser) [reply README.md:37]: I feel the first paragraph is somehow restarting what was already at the top of the README? Maybe just use the 2nd paragraph only?

Comment (PapyChacal) [reply README.md:37]: Oh I missed your comment while writing mine, I agree, I'll push without the first paragraph and see.

Comment (tobiasgrosser) [reply README.md:37]: Nice. That looks good new from my perspective. I wonder what @georgebisbas thinks

**Review by math-fehr — APPROVED** [review]
Nice! Looks great to me!

File: README.md [hunk 2]
```diff
@@ -35,11 +35,13 @@ pip install xdsl
...
 
-- [A simple introduction](docs/tutorial.ipynb)
-- [A DSL for defining new IRs](docs/irdl.ipynb)
-- [Connecting xDSL with MLIR](docs/mlir_interoperation.md)
+The following tutorials will respectively present xDSL basic concepts, how to
```
[the remark this answers is not in the document]
Comment (PapyChacal) [reply README.md]: Done!

**Pull request merged**